.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -38,8 +38,11 @@ body:
         - OS Version: [e.g. Ubuntu 24.04.2 LTS]
         - PostgreSQL Version: [e.g. 15.1]
         - PGroonga Version: [e.g. 12.1.0]
+           - You can check the version by running:
+               SELECT extversion FROM pg_catalog.pg_extension WHERE extname = 'pgroonga';
```

```suggestion
           - You can check the version by running:
             SELECT extversion FROM pg_catalog.pg_extension WHERE extname = 'pgroonga';
```

Did we need indentation here?